uportal-war/src/main/data/required_entities/entity-type/org.apereo.portal.security.IPerson.entity-type.xml

```diff
@@ -20,6 +20,6 @@
 
 -->
 <entity-type script="classpath://org/jasig/portal/io/import-entity-type_v3-2.crn">
-  <name>org.apereo.portal.security.IPerson</name>
+  <name>org.jasig.portal.security.IPerson</name>
```

Comment:
same here, shouldn't these be changing from `jasig` to `apereo`, not vice verse?